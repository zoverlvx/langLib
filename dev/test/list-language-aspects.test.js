import React from "react";
import ListLanguageAspects from "../js/components/list-language-aspects";
import toJson from "enzyme-to-json";
import {shallow} from "enzyme";

const component = shallow(<ListLanguageAspects />);
const tree = toJson(component);

describe("<ListLanguageAspects />", () => {
	it("renders 1 <ListLanguageAspects /> component", () => {
		console.log(tree);
		expect(component).toHaveLength(1);
	});

	it("provides a snapshot of ListLanguageAspects' tree", () => { 
		expect(tree).toMatchSnapshot();
	});
});

//What ought to be tested