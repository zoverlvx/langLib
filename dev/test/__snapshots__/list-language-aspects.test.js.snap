// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListLanguageAspects /> provides a snapshot of ListLanguageAspects' tree 1`] = `
<Route
  render={[Function]}
/>
`;
